python/ray/rllib/train.py

```diff
@@ -18,6 +18,7 @@
     description=("Train a reinforcement learning agent."))
 parser.add_argument("--env", required=True, type=str)
 parser.add_argument("--alg", required=True, type=str)
+parser.add_argument("--config", default="{}", type=str)
```

Comment:
You can already do this:
```
python train.py --env Humanoid-v1 --alg PolicyGradient --config '$(< config.json)'
```

I think it is preferable to not have to have too many files flying around, but I'm happy to be convinced here if lots of people have different opinions!